### PR TITLE
Fixed RawChicken::getAdditionalEffects sometimes returns void

### PR DESCRIPTION
**Pull request by SOF3**

### CODE REVIEW

File: src/pocketmine/item/RawChicken.php [hunk 1]
```diff
@@ -40,6 +40,7 @@ public function getAdditionalEffects() : array{
 		if(mt_rand(0, 9) < 3){
 			return Effect::getEffect(Effect::HUNGER)->setDuration(600);
 		}
```
Comment (dktapps) [inline src/pocketmine/item/RawChicken.php]: Well, this is never going to return an array anyway...

Comment (iPocket) [inline src/pocketmine/item/RawChicken.php]: Then, empty string or bool?

Comment (iPocket) [inline src/pocketmine/item/RawChicken.php]: https://github.com/pmmp/PocketMine-MP/pull/4/files#diff-561903db00d738e771f0fb58714e2084L39

Comment (dktapps) [inline src/pocketmine/item/RawChicken.php]: It should be an array to conform with the Food declaration. https://github.com/pmmp/PocketMine-MP/blob/master/src/pocketmine/item/Food.php#L50

Comment (thebigsmileXD) [inline src/pocketmine/item/RawChicken.php]: Damn, he is back to CS code xD

**Comment by SOF3** [comment]
Fixed, please squash.

File: src/pocketmine/item/RawChicken.php (hunk 1 above)
Comment (archie426) [inline src/pocketmine/item/RawChicken.php]: CS code?!

Comment (archie426) [inline src/pocketmine/item/RawChicken.php]: Isnt that VS code, I have never heard of CS before, which do you recommend?

Comment (dktapps) [inline src/pocketmine/item/RawChicken.php]: This isn't CS code. It's @PEMapModder 's code.

Comment (thebigsmileXD) [inline src/pocketmine/item/RawChicken.php]: The mess of no-array-food was CS

Comment (iPocket) [inline src/pocketmine/item/RawChicken.php]: @archie426 dump, he was talking about ClearSky code...

**Pull request merged**